conf/ocsci/aws_minimal_deployment.yaml

```diff
@@ -0,0 +1,7 @@
+DEPLOYMENT:
+  allow_lower_instance_requirements: true
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_instance_type: 'm5.2xlarge'
```

Comment:
This is matching the minimum requirement of CPU only , right ?

Reply:
yes. minimal requirment per worker node is 8CPU